src/elements/helpers/Preview.tsx

```diff
@@ -24,6 +43,19 @@ const getDocHeight = (doc: Document | undefined) => {
   return undefined;
 };
 
+const isMessageData = (data: unknown): data is MessageData => {
+  if (
```

Comment:
IIUC could just return this set of comparisons directly w/o an if statement, but you might prefer this for readability – not a strongly held opinion.

Reply:
That would be neater - have implemented now.